src/MBrace.Streams/CloudStreams.fs

```diff
@@ -703,5 +703,54 @@ module CloudStream =
         sources
         |> ofCloudFiles CloudFileReader.ReadLines
         |> flatMap Stream.ofSeq
-        
+
+    /// <summary>
+    /// Constructs a CloudStream of lines from a path.
+    /// </summary>
+    /// <param name="path">The path to the text file.</param>
+    let ofTextFile (path : string) : CloudStream<string> =
```

Comment:
Thanks Don! I'll rename it to ofTextFileByLine. Btw the Spark uses this 
def textFile(path: String, minSplits: Int = defaultMinSplits): RDD[String]
https://spark.apache.org/docs/0.6.2/api/core/spark/SparkContext.html